odra-casper/backend/src/contract_env.rs

```diff
@@ -144,6 +144,6 @@ pub fn verify_signature(
 }
 
 /// Creates a hash of the given input. Uses default hash for given backend.
-pub fn hash<T: AsRef<[u8]>>(input: T) -> Vec<u8> {
+pub fn hash<T: AsRef<[u8]>>(input: T) -> alloc::vec::Vec<u8> {
```

Comment:
Avoid `alloc::vec::Vec<u8>`, import it instead.

Reply:
Done